### PR TITLE
Add note that raw endpoint doesn't send heartbeats

### PR DESCRIPTION
**Pull request by njam**
Took me a while to understand this was the reason that my connections would "die".

Wdyt? Maybe it helps someone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,6 +309,8 @@ this url. This can be a command line client, external application,
 third party code or even a browser (though I don't know why you would
 want to do so).
 
+Note that on this endpoint the server will *not send any heartbeat packets*.
```
Comment (brycekahle) [inline README.md]: Maybe reword to "Note: this endpoint will *not send any heartbeat packets*."?

**Comment by njam** [comment]
@brycekahle adjusted, please review
